### ISSUE TITLE
Who will keep the wheel files current with tensorflow releases?

### ISSUE THREAD

**Issue by kaisark** [jetsonhacks/installTensorFlowJetsonTX#8]
Who will keep the wheel files current with tensorflow releases?

TF 1.7 is out...

**Comment by saikishor** [comment]
Thanks for sharing your file @kaisark

**Comment by MerlotQ** [comment]
@kaisark Is it a gpu version of tensorflow 1.5?

**Comment by saikishor** [comment]
@MerlotQ Yes it is a GPU version. Usually, it should be of GPU when we talk about Jetson, as there is no point of running a CPU version on it.

**Comment by mprat** [comment]
@kaisark  Thanks for sharing this. You should submit a PR against this repo with the new wheel file - maybe others can find it easier that way!

**Comment by kaisark** [comment]
@MerlotQ Yes, it was GPU version using build scripts from Jetsonhacks.

It looks like there are folks out there that are creating new TF wheel files for Jetson TX1/TX2, but you might need to upgrade to CUDA 9 via a Jetpack flash. I provided the related links below.

https://devtalk.nvidia.com/default/topic/1031300/jetson-tx2/tensorflow-1-11-0-wheel-with-jetpack-3-3/1

https://github.com/peterlee0127/tensorflow-nvJetson
